app/code/Magento/CompareListGraphQl/composer.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "magento/module-compare-list-graph-ql",
+    "version": "1.0.0",
```

Comment:
Hello @Usik2203 
Thank you for your contribution

Please remove unneeded element `version`
I think it can help you resolving the problem with static tests.
```suggestion
```

Reply:
**Done**